source/MaterialXCore/Node.h

```diff
@@ -160,6 +160,10 @@ class MX_CORE_API Node : public InterfaceElement
     /// If the input already exists on the node it will just be returned.
     InputPtr addInputFromNodeDef(const string& name);
 
+    /// Add inputs based on the corresponding associated node definition.
+    /// If the inputs already exists on the node it will just be returned.
```

Comment:
It looks like this second sentence isn't needed in the comment, as it only really applies to the original addInputFromNodeDef method.